k8s/regions/frankfurt/test.yaml

```diff
@@ -0,0 +1,147 @@
+            liveness:
+                failure_threshold: 2
+                initial_delay_seconds: 5
+                period_seconds: 5
+                success_threshold: 1
+                timeout_seconds: 5
+app:
+    allowed_cidr_nets: SECRET
+    allowed_hosts: ".sumo.mozit.cloud,support-dev.allizom.org"
+    aws_access_key_id: SECRET
+    aws_s3_custom_domain: SECRET
+    aws_secret_access_key: SECRET
+    aws_storage_bucket_name: SECRET
+    broker_url: SECRET
+    cache_url: SECRET
+    celery_already_eager: False
+    csrf_cookie_secure: True
+    database_url: SECRET
+    default_from_email: "notifications-dev@support.mozilla.org"
+    db_conn_max_age: 60
+    debug: False
+    dms_auto_archive_old_questions: SECRET
+    dms_cache_most_unhelpful_kb_articles: SECRET
+    dms_calculate_csat_metrics: SECRET
+    dms_clear_expired_auth_tokens: SECRET
+    dms_cohort_analysis: SECRET
+    dms_enqueue_lag_monitor_task: SECRET
+    dms_fix_current_revisions: SECRET
+    dms_generate_missing_share_links: SECRET
+    dms_process_exit_surveys: SECRET
+    dms_purge_hashes: SECRET
+    dms_rebuild_kb: SECRET
+    dms_reindex: SECRET
+    dms_reindex_users_that_contributed_yesterday: SECRET
+    dms_reload_question_traffic_stats: SECRET
+    dms_reload_wiki_traffic_stats: SECRET
+    dms_remove_expired_registration_profiles: SECRET
+    dms_report_employee_answers: SECRET
+    dms_send_weekly_ready_for_review_digest: SECRET
+    dms_send_welcome_emails: SECRET
+    dms_survey_recent_askers: SECRET
+    dms_update_contributor_metrics: SECRET
+    dms_update_l10n_contributor_metrics: SECRET
+    dms_update_l10n_coverage_metrics: SECRET
+    dms_update_l10n_metric: SECRET
+    dms_update_product_details: SECRET
+    dms_update_top_contributors: SECRET
+    dms_update_weekly_votes: SECRET
+    k8s_domain: "frankfurt.mozit.cloud"
+    email_host: SECRET
+    email_host_user: SECRET
+    email_host_password: SECRET
+    email_logging_real_backend: "django.core.mail.backends.smtp.EmailBackend"
+    email_port: 25
+    email_subject_prefix: "[support-dev] "
+    email_use_tls: True
+    enable_admin: True
+    enable_whitenoise: True
+    enforce_host: "dev.sumo.mozit.cloud,support-dev.allizom.org"
+    engage_robots: False
+    es_index_prefix: "sumo_dev"
+    es_http_auth: SECRET
+    es_live_indexing: True
+    es_use_ssl: True
+    es_urls: SECRET
+    ga_account: SECRET
+    ga_key: SECRET
+    ga_profile_id: SECRET
+    gtm_container_id: "UA-36116321-2"
+    media_url: "https://user-media-dev-cdn.itsre-sumo.mozilla.net/"
+    new_relic_app_name: "sumo-dev-frankfurt"
+    new_relic_license_key: SECRET
+    pipeline_enabled: True
+    port: 8000
+    read_only: False
+    redis_default_url: SECRET
+    redis_helpfulvotes_url: SECRET
+    run_migrate: "true"
+    secret_key: SECRET
+    sentry_dsn: SECRET
+    session_cookie_secure: True
+    stage: True
+    static_url: "https://static-media-dev-cdn.itsre-sumo.mozilla.net/static/"
```

Comment:
s/dev/test?

Reply:
We'll re-use dev CDN endpoints as per conversation.